### PR TITLE
CAKE-5215 | Add an automated way of mass-setting wgVars

### PR DESCRIPTION
**Pull request by vforge**
Running this on dev:
```
$> docker run -it --rm -e "SERVER_ID=177" -e "WIKIA_DEV_DOMAIN=$WIKIA_DEV_DOMAIN" -e "WIKIA_ENVIRONMENT=$WIKIA_ENVIRONMENT" -e "WIKIA_DATACENTER=$WIKIA_DATACENTER" -e "HOSTNAME_OVERRIDE=`hostname`" -v "$HOME/app":/usr/wikia/slot1/current/src:ro -v "$HOME/config":/usr/wikia/slot1/current/config:ro -v "$HOME/cache":/usr/wikia/slot1/current/cache/messages artifactory.wikia-inc.com/platform/php-wikia-devbox:latest php maintenance/wikia/MassSetWikiFactoryValues.php -d maintenance/wikia/MassSetWikiFactoryValues.dev.csv
```

To run this script FOR REAL you need to remove `-d` param

### CODE REVIEW

File: maintenance/wikia/MassSetWikiFactoryValues.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+# https://docs.google.com/spreadsheets/d/1vGuJEZ0ncVQSXTrABM3YYHg2P531sWZosgNsq-uMy5M/edit#gid=2073314697
+
+class MassSetWikiFactoryValues extends Maintenance {
+	protected $saveChanges  = false;
+	
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = 'Sets Wiki Factory variable values based on CSV';
+		$this->addArg( 'file', 'CSV file with the list of wikis and vars' );
+		$this->addOption( 'saveChanges', 'Change the wiki values for real.', false, false, 'd' );
```
Comment (mklucsarits) [inline maintenance/wikia/MassSetWikiFactoryValues.php]: Maybe use a "--dry-run" parameter to make it more clear?

File: maintenance/wikia/MassSetWikiFactoryValues.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	}
+	
+	private function readFromCSV() {
+		$fileName = $this->getArg( 0 );
+		
+		$fileHandle = fopen( $fileName, 'r' );
```
Comment (mklucsarits) [inline maintenance/wikia/MassSetWikiFactoryValues.php:21]: Let's validate the fileHandle and exit if it's bad.

Comment (vforge) [reply maintenance/wikia/MassSetWikiFactoryValues.php:21]: d3a5b5e

File: maintenance/wikia/MassSetWikiFactoryValues.php (hunk 1 above)
Comment (vforge) [reply maintenance/wikia/MassSetWikiFactoryValues.php]: d3a5b5e

**Review by JoshuaRogan — APPROVED** [review]

**Pull request merged**